glances/plugins/cpu/__init__.py

```diff
@@ -271,7 +271,7 @@ def update_snmp(self):
                 return self.stats
 
             # Convert SNMP stats to float
-            for key in iterkeys(stats):
+            for key in .keys(stats):
```

Comment:
Rather `stats.keys()`: `iterkeys` in Python 2 is replaced by [the keys method](https://docs.python.org/3/library/stdtypes.html#dict.keys) in Python 3.